src/components/SignIn/SignInForm.tsx

```diff
@@ -109,7 +110,7 @@ const SignInForm = (): React.JSX.Element => {
 			</Button>
 
 			<div>
-				<a href={`${apiUrl}forgot-passsword`} className='do-not-remember-password'>I don't remember my password</a>
+				<a href={`${FE_URL}forgot-passsword`} className='do-not-remember-password'>I don't remember my password</a>
```

Comment:
```suggestion
				<a href="/forgot-passsword" className='do-not-remember-password'>I don't remember my password</a>
```

Reply:
I tried implenting this on sign-in and sign-up and it doesn't work.